<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>528</width>
    <height>422</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLCDNumber" name="lcdStopwatch">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>191</width>
      <height>61</height>
     </rect>
    </property>
    <property name="smallDecimalPoint">
     <bool>false</bool>
    </property>
    <property name="digitCount">
     <number>6</number>
    </property>
    <property name="segmentStyle">
     <enum>QLCDNumber::Filled</enum>
    </property>
    <property name="intValue" stdset="0">
     <number>0</number>
    </property>
   </widget>
   <widget class="QLCDNumber" name="lcdTimer">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>200</y>
      <width>191</width>
      <height>61</height>
     </rect>
    </property>
    <property name="smallDecimalPoint">
     <bool>false</bool>
    </property>
    <property name="digitCount">
     <number>6</number>
    </property>
    <property name="segmentStyle">
     <enum>QLCDNumber::Filled</enum>
    </property>
    <property name="intValue" stdset="0">
     <number>0</number>
    </property>
   </widget>
   <widget class="QProgressBar" name="pbTimer">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>270</y>
      <width>191</width>
      <height>23</height>
     </rect>
    </property>
    <property name="value">
     <number>24</number>
    </property>
   </widget>
   <widget class="QPushButton" name="bStopwatchNew">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>86</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>New</string>
    </property>
   </widget>
   <widget class="QPushButton" name="bStopwatchPause">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>100</y>
      <width>86</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>Pause</string>
    </property>
   </widget>
   <widget class="QPushButton" name="bTimerSave">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>140</y>
      <width>86</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>Save</string>
    </property>
   </widget>
   <widget class="QLCDNumber" name="lcdClock">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>350</y>
      <width>191</width>
      <height>61</height>
     </rect>
    </property>
    <property name="smallDecimalPoint">
     <bool>false</bool>
    </property>
    <property name="digitCount">
     <number>6</number>
    </property>
    <property name="segmentStyle">
     <enum>QLCDNumber::Filled</enum>
    </property>
    <property name="intValue" stdset="0">
     <number>0</number>
    </property>
   </widget>
   <widget class="QLineEdit" name="tTimerTime">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>200</y>
      <width>181</width>
      <height>26</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="bTimerReset">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>270</y>
      <width>86</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>Reset</string>
    </property>
   </widget>
   <widget class="QPushButton" name="bTimerStart">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>230</y>
      <width>86</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>Start</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>521</width>
      <height>171</height>
     </rect>
    </property>
    <property name="title">
     <string>Stopwatch</string>
    </property>
    <widget class="QPlainTextEdit" name="tTimeout">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>30</y>
       <width>141</width>
       <height>131</height>
      </rect>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPlainTextEdit" name="tTimeValue">
     <property name="geometry">
      <rect>
       <x>360</x>
       <y>30</y>
       <width>61</width>
       <height>131</height>
      </rect>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
     <property name="plainText">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>170</y>
      <width>521</width>
      <height>171</height>
     </rect>
    </property>
    <property name="title">
     <string>Timer</string>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>30</y>
       <width>121</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Note: Write time&lt;/p&gt;&lt;p&gt;in minutes!&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="scaledContents">
      <bool>false</bool>
     </property>
     <property name="indent">
      <number>-1</number>
     </property>
    </widget>
    <widget class="QCheckBox" name="bBeep">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>130</y>
       <width>181</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Beep when done</string>
     </property>
    </widget>
    <widget class="QPushButton" name="bTimerPause">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>60</y>
       <width>86</width>
       <height>26</height>
      </rect>
     </property>
     <property name="text">
      <string>Pause</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="bStopwatchSave">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>30</y>
      <width>86</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>Save Output</string>
    </property>
   </widget>
   <zorder>groupBox</zorder>
   <zorder>groupBox_2</zorder>
   <zorder>lcdTimer</zorder>
   <zorder>pbTimer</zorder>
   <zorder>bStopwatchNew</zorder>
   <zorder>bStopwatchPause</zorder>
   <zorder>bTimerSave</zorder>
   <zorder>lcdClock</zorder>
   <zorder>tTimerTime</zorder>
   <zorder>bTimerReset</zorder>
   <zorder>bTimerStart</zorder>
   <zorder>bStopwatchSave</zorder>
   <zorder>lcdStopwatch</zorder>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
